sessionctx/variable/sysvar.go

```diff
@@ -683,6 +683,7 @@ var defaultSysVars = []*SysVar{
 	{ScopeSession, TiDBForcePriority, mysql.Priority2Str[DefTiDBForcePriority]},
 	{ScopeSession, TiDBEnableRadixJoin, BoolToIntStr(DefTiDBUseRadixJoin)},
 	{ScopeSession, TiDBCheckMb4ValueInUtf8, BoolToIntStr(config.GetGlobalConfig().CheckMb4ValueInUtf8)},
+	{ScopeSession, TiDBSlowQueryFile, ""},
```

Comment:
If TiDBSlowQueryFile can never be modified, the scope should be ScopeNone.

Reply:
If log rotate enable, may be need to parse yesterday log, or parse multi-log files, then can set TiDBSlowQueryFile to parse special log file, so I think this variable should be modified as session scope.